tests/sentry/api/endpoints/test_organization_artifactbundle_assemble.py

```diff
@@ -141,6 +141,86 @@ def test_assemble_with_invalid_projects(self):
         assert response.status_code == 400, response.content
         assert response.data["error"] == "One or more projects are invalid"
 
+    def test_assemble_with_valid_project_slugs(self):
```

Comment:
subjective nit: maybe this test isn't needed b/c we're using project slug already for everything else? This is prob more subjective tho cuz maybe it's good to explicitly test anyways. Up to you